---
title: "Research"
---

## <font color='#3498DB'> Phylogenetically informative data from next-generation sequencing </font>

We have long expected that complete genomic information will provide us with the tools to determine relationships among organisms, from strains of pathogens to the whole tree of life. However, while whole genome sequencing is now possible, extracting information from these data remains challenging. I have developed software (SISRS) to easily identify phylogenetically informative data from raw next-generation sequence data. We have used SISRS data to show that non-coding loci provide more overall signal and a higher proportion of phylogenetic signal compared to coding loci, and different types of loci (e.g. coding vs. introns) have surprisingly consistent levels of information across time scales. 

### Related publications

[Literman, R. and R.S. Schwartz. 2021. Genome-scale profiling reveals higher proportions of phylogenetic signal in non-coding data. Molecular Biology and Evolution.](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab026/6126411)

[Harkins, K.M., R.S. Schwartz, R.A. Cartwright, and A.C. Stone. 2016. Phylogenomic reconstruction supports supercontinent origins for Leishmania. Infection, Genetics and Evolution 38: 101-109.](https://www.sciencedirect.com/science/article/abs/pii/S1567134815300605)

[Schwartz, R.S., K.M Harkins, A.C. Stone, and R.A. Cartwright. 2015. A composite genome approach to identify phylogenetically informative data from next-generation sequencing. BMC Bioinformatics. 16:193.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0632-y)

*This research is/was funded by NSF grant DBI-1942273 to R. Schwartz, NSF grant DBI-1356548 to R. Cartwright, funding from URI, and an NSF postdoctoral fellowship to R. Literman.*

<hr>

## <font color='#3498DB'> Estimating diversification rates from phylogenies </font>

### Related publications

[Schwartz, R.S. and R.L. Mueller. Branch length estimation and divergence dating: estimates of error in Bayesian and maximum likelihood frameworks. BMC evolutionary biology 10: 5.](https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-10-5)

*This research was funded by an NSF postdoctoral fellowship to R. Schwartz.*

## <font color='#3498DB'> Copy number variation in genomes </font>
